packages/helix-shared-config/test/markupconfig.test.js

```diff
@@ -67,7 +67,7 @@ describe('Markup Config Loading', () => {
       if (e instanceof AssertionError) {
         throw e;
       }
-      assert.equal(e.message, 'data/markup/images-in-gallery must have required property \'match\', data/markup/last-section must NOT have additional properties');
+      assert.equal(e.message, 'Invalid configuration:\nMarkup Mapping must have required property \'match\'\nMarkup Mapping has unknown property \'invalid\'\n\ndata/markup/images-in-gallery must have required property \'match\', data/markup/last-section must NOT have additional properties');
```

Comment:
Messages to compare against got longer - may be use a different message formatter when called via `SchemaDerivedConfig`?

Reply:
It's now listing all errors, not just the first. This is useful.